app/views/facility_statements/new.html.haml

```diff
@@ -9,7 +9,10 @@
   = render :partial => 'shared/transactions/top', :locals => { :tab => 'statements' }
```

Comment:
While we're in this file, can we clean up the style? Hash syntax, double quotes, prefer `render "shared/transactions/top", tab: "statements"` to using the `partial` option.